Makefile

```diff
@@ -20,4 +26,10 @@ preview-build:
 		--buildDrafts \
 		--buildFuture \
 		--baseURL $(DEPLOY_PRIME_URL) \
-		--minify
\ No newline at end of file
+		--minify
+
+docker-image:
+	$(DOCKER) build . --tag $(DOCKER_IMAGE)
```

Comment:
Better to just say `docker` here to avoid indirection, and `tikv/website` for proper namespacing.